include/uacpi/acpi.h

```diff
@@ -1003,6 +1003,66 @@ UACPI_PACKED(struct acpi_ecdt {
 })
 UACPI_EXPECT_SIZEOF(struct acpi_ecdt, 65);
 
+#define ACPI_DBG2_TYPE_SERIAL 0x8000
+#define ACPI_DBG2_TYPE_1394 0x8001
+#define ACPI_DBG2_TYPE_USB 0x8002
+#define ACPI_DBG2_TYPE_NET 0x8003
```

Comment:
I get that other types are less common, but since this is a generic library for everyone, we might as well add all of them, from a quick glance there isn't that many of them